sources/Application/Views/PhraseView.cpp

```diff
@@ -595,34 +595,48 @@ void PhraseView::pasteClipboard() {
     uint *src5=clipboard_.cmd2_ ;
     ushort *dst6=viewData_->song_->phrase_->param2_+16*viewData_->currentPhrase_ ;
     ushort *src6=clipboard_.param2_ ;
-    
+
+	uint* noCmd = (uint*) -1;
```

Comment:
why do force-cast assign negative values to variables that are typed as unsigned?

Reply:
I just wanted a preset for "nothing in here don't worry about it"